src/basho_bench.erl

```diff
@@ -346,3 +368,70 @@ get_test_dir() ->
         undefined -> error(unset_test_dir);
         {ok, TestDir} -> TestDir
     end.
+
+master_node() ->
+    case catch basho_bench_config:get(master_node, undefined) of
+        {'EXIT', {noproc, _}} ->
+            case init:get_argument(master_node) of
+                {ok, [[Node]]} -> list_to_atom(Node);
+                _ -> node()
+            end;
+        undefined ->
+            node();
+        Res ->
+            Res
+    end.
+
+node_count() ->
+    case catch basho_bench_config:get(node_count, undefined) of
+        {'EXIT', {noproc, _}} ->
```

Comment:
Do we have a config reading race condition at startup or something?

Reply:
Yeah if you call `node_count()` before the `basho_bench_config` process has been spawned then we can't load it from the config values and so we fall back to loading it from init.

Reply:
Hmmmm... I think it makes sense to update the `undefined` clause to also check the init args values, rather than just defaulting to `node()`. I'll update that.

Reply:
Fixed in 183fb57.